server.js

```diff
@@ -5,7 +5,7 @@ const path = require("path");
 const fetch = require("node-fetch");
 
 const CLIENT_ID = process.env.REACT_APP_CLIENT_ID;
-const CLIENT_SECRET = process.env.REACT_APP_CLIENT_SECRET;
+const CLIENT_SECRET = process.env.TINK_CLIENT_SECRET;
```

Comment:
Will other than `REACT_APP`-prefixed env variable be available in the app?

from docs:
Note: You must create custom environment variables beginning with REACT_APP_. Any other variables except NODE_ENV will be ignored to avoid accidentally exposing a private key on the machine that could have the same name. Changing any environment variables will require you to restart the development server if it is running.

https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
> Will other than REACT_APP-prefixed env variable be available in the app?

Not for the client. But e.g. `TINK_CLIENT_SECRET` will still be available for the express server we're relying on in this implementation.